node/service/src/service.rs

```diff
@@ -527,7 +530,7 @@ pub fn frequency_dev_instant_sealing(
 			.import_notification_stream()
 			.filter(move |_| futures::future::ready(is_instant))
 			.map(|_| sc_consensus_manual_seal::rpc::EngineCommand::SealNewBlock {
-				create_empty: false,
+				create_empty: true,
```

Comment:
Instant sealing doesn't need to create empty blocks as far as I know.

```suggestion
				create_empty: false,
```

Reply:
It's required.  When set to false, transactions fail with this error:

`2023-05-08 16:33:52.325 ERROR tokio-runtime-worker sc_consensus_manual_seal::rpc: Instant Seal encountered an error: Transaction pool is empty, set create_empty to true, if you want to create empty blocks    
`